book/cli/reth/debug/execution.md

```diff
@@ -226,6 +226,11 @@ Networking:
 
           [default: 25600]
 
+      --docker-if <DOCKER>
+          Name of docker non-host interface used to communicate with peers.
+
+          If flags is set, but no value is passed, the default interface `eth0` is tried.
```

Comment:
imo this is unrelated this pr, we already have --nat:extip

unclear how this is helpful here, let's think about this separately